### ISSUE TITLE
False positives for `is_pydantic_model`

### ISSUE THREAD

**Issue by Viicos** [Viicos/flake8-pydantic#9]
The following checks:
- `_has_field_function`
- `_has_pydantic_method`

Raises false positives with the following code:

- Using `django-import-export`:

```python
class SomeResource(resources.ModelResource):
    user = Field(attribute="user")
```

- When a `model_*` method is defined.
